src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -392,7 +392,7 @@ await foreach (var response in requests)
                 Methods.CodeActionResolveName,
                 SupportsCSharpCodeActions,
                 codeAction,
-                cancellationToken).ConfigureAwait(false);
+                CancellationToken.None).ConfigureAwait(false);
```

Comment:
This is surprising...

Reply:
Ooops I think @dibarbet accidentally left this in from debugging. He was seeing aggressive cancellation without this

Reply:
yup accidental debugging code